src/main/resources/templates/xbox.html

```diff
@@ -9,13 +9,13 @@
 <!-- Bootstrap CSS -->
 <link rel="stylesheet"
 	href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
-	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
-	crossorigin="anonymous">
-<title>xbox Game</title>
+	data-integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
+	data-crossorigin="anonymous">
+<title>XBox Games</title>
 </head>
 <body>
 	<div class="container">
-		<h4>Xbox Games</h4>
+		<h4>XBox Games</h4>
```

Comment:
Keep the title of headline consistent with the page title to help users avoid being misled (unsure about if needed)